src/basic/copy.c

```diff
@@ -114,9 +112,9 @@ int copy_bytes(int fdf, int fdt, uint64_t max_bytes, bool try_reflink) {
 
                 /* As a fallback just copy bits by hand */
                 {
-                        uint8_t buf[m];
+                        uint8_t buf[MIN(m, COPY_BUFFER_SIZE)];
 
-                        n = read(fdf, buf, m);
+                        n = read(fdf, buf, sizeof buf);
```

Comment:
It's like `return` :) It is slightly shorter without the extra parens, and I think it's more readable (for short arguments).